parse/parse.go

```diff
@@ -109,6 +110,7 @@ func Package(path string, files []string) (*PkgInfo, error) {
 	}
 
 	setDefault(p, pi, info)
+	setAliases(p, pi, info)
```

Comment:
one more thing, can you sanity check that all the aliases refer to valid targets?   Thanks!